serf/serf.go

```diff
@@ -1438,6 +1453,33 @@ func (s *Serf) resolveNodeConflict() {
 	}
 }
 
+//handlePrune is a simplified version of Reap() which only runs when the prune
+//flag is set
+func (s *Serf) handlePrune(old []*memberState, m *memberState) []*memberState {
+	// Delete from members
+	delete(s.members, m.Name)
+
+	// Tell the coordinate client the node has gone away and delete
+	// its cached coordinates.
+	if !s.config.DisableCoordinates {
+		s.coordClient.ForgetNode(m.Name)
+
+		s.coordCacheLock.Lock()
+		delete(s.coordCache, m.Name)
+		s.coordCacheLock.Unlock()
+	}
+
+	// Send an event along
+	s.logger.Printf("[INFO] serf: EventMemberReap: %s", m.Name)
```

Comment:
Is it worth making this log different to indicate it was a force reap not a time-based one? Maybe something like:
```suggestion
	s.logger.Printf("[INFO] serf: EventMemberReap (forced): %s", m.Name)
```

Reply:
I totally dig this. I was thinking if wondering if I _should_ do something like that and was hoping that someone would comment on it!